### DISCUSSION TITLE
feedback: release 2.0.0-alpha-0

### DISCUSSION THREAD

**Discussion by jeremylong-snc** [CycloneDX/cyclonedx-gradle-plugin#558] [General]
@skhokhlov via the grapevine I was asked to provide feedback, as such I'm opening this issue as there is no discussion board for the project even though this isn't really an "issue".

**The initial testing of the alpha release looks good.** ~~We did run into an exception due to how a mutli-project gradle **android** build was setup:~~

~~[removed stacktrace]~~

I may suggest 1-2 other minor improvements via PRs - but they should not hold up a release. Although for my purposes I will need #546 merged - but if it doesn't make it I can just build the project locally and use my version until the build-system metadata is added.

**Comment by jeremylong-snc** [comment]
The alpha release looks good! I updated my comment above after researching the stacktrace during bom generation - it is nothing the plugin should "fix".

**Comment by skhokhlov** [comment]
@jeremylong-snc great to hear! Please feel free to raise issues / send PRs for your improvements. FYI, I expect version 2.0.0 to be released on January 16th.
